### PR TITLE
Minimal splunk-hec destination docs from the release notes

### PR DESCRIPTION
**Pull request by fekete-robert**

### CODE REVIEW

**Comment by alltilla** [comment]
IMO `sh` syntax highlighting would produce a bit better results for syslog-ng config snippets. E.g. `raw` being in a different color seems weird to me.

#### c
image

#### sh
image

File: content/docs/configure-destination/splunk/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+destination d_splunk_hec_raw {
+  splunk-hec-raw(
+    url("https://localhost:8088")
+    token("70b6ae71-76b3-4c38-9597-0c5b37ad9630")
+    channel("05ed4617-f186-4ccd-b4e7-08847094c8fd")
+  );
+};
+```
+
+The options of the `splunk-hec-raw()` destination are similar to the `splunk-hec-event()` destination, but it has a mandatory `channel()` option, which accepts a GUID as a literal string that differentiates the data from different clients.
```
Comment (alltilla) [inline content/docs/configure-destination/splunk/_index.md]: Maybe we could mention this Splunk doc: https://docs.splunk.com/Documentation/Splunk/9.0.4/Data/FormateventsforHTTPEventCollector#Channel_identifier_header

What is not clear from the Splunk doc IMO, that this GUID is not generated by Splunk, the user has to generate one for themselves and use that. There is no need to add the GUID to Splunk beforehand, etc...

**Review by alltilla — APPROVED** [review]
2 small comments, LGTM otherwise :)

File: content/docs/configure-destination/splunk/_index.md (hunk 1 above)
Comment (fekete-robert) [reply content/docs/configure-destination/splunk/_index.md]: Is it possible to generate the GUID with a template or template function in syslog-ng (like $HOST), or that doesn't make sense?

Comment (alltilla) [reply content/docs/configure-destination/splunk/_index.md]: I don't think that would be useful, as the GUID should identify the client. If it is always generated (once runtime or once per message), it is good for uniqueness but not for identification, as the user cannot have a GUID <-> client mapping.

Comment (alltilla) [reply content/docs/configure-destination/splunk/_index.md]: How I imagine this is that the user generates a GUID, configures a client and writes this mapping to a "sticky note", so when they check the messages in splunk, they can work out the client by matching the messages's channel GUID with the one on the sticky note.

**Review by alltilla — APPROVED** [review]

**Comment by fekete-robert** [comment]
Ok, thanks!

**Pull request merged**